x-pack/plugins/security_solution/public/common/components/header_page/index.tsx

```diff
@@ -59,7 +60,7 @@ interface BackOptions {
   href: LinkIconProps['href'];
   text: LinkIconProps['children'];
   dataTestSubj?: string;
-  pageId: SecurityPageName;
+  pageId: SiemNavTabKey;
```

Comment:
There was a type problem because now we have pages (used by the deepLinks) that are not in the main navigation (detections, administration, policies...), and I think it is okay, not all pages have to be in the main nav.
But some functions such as `useFormatUrl` still needs a navigation tab key, as it cheks properties.
Anyway, I agree this types can be improved a lot, as I told Xavier here https://github.com/elastic/kibana/pull/103275/files#r659861940
we can do it in a following PR